yaksh/documentation/moderator_docs/creating_course.rst

```diff
@@ -6,47 +6,73 @@ For students to take a quiz, it is imperative for the moderator to create a cour
+        * Quiz Name
+            Clicking on the quiz name will let you edit the quiz.
+        * Question Paper
+            Click on the **Add** link to create a Question Paper for associated Quiz.
+            If a question paper is already created, click on the Question Paper link to edit question paper.
+        * Add Teacher
+            Clicking on Add teacher can let you add teachers for the course. The teachers can edit and modify only the specific course that are allotted to them.
+        * Teachers added to the course
+            This shows all the teachers added to a particular course.
+        * Download CSV for the entire course
+            This downloads the CSV file containing the performance of all students in every quiz for a given course.
+        * Edit Course
+            Clicking on the edit course button will let you edit the details of an existing course.
 
-	 .. image:: ../images/course_features.jpg
+Features in Course Details
+--------------------------
 
-	 This page shows all the courses created by a moderator and all the courses allotted to a moderator.
+    Click on a given course name to go to the course details page.
 
-	 The following features are available for courses
+    .. image:: ../images/course_details_features.jpg
 
-	 	* Course Name
-	 		Clicking on course name link will display all the enrolled, rejected and requested students list. Moderator can accept or reject the student.	 	
-	 	* Quiz Name
-	 		Clicking on the quiz name will let you edit the quiz.
-	 	* Question Paper
-	 		Click on the **Add** link to create a Question Paper for associated Quiz.
-	 		If a question paper is already created, click on the Question Paper link to 
-	 		edit question paper.
-	 	* Add Teacher
-	 		Clicking on Add teacher can let you add teachers for the course. The teachers can edit and modify only the specific course that are allotted to them.
-	 	* Teachers added to the course
-	 		This shows all the teachers added to a particular course.
+    Following are the features for course details -
 
+        * Requests
+            This is a list of students who have requested to be enrolled in the course. Moderator can enroll or reject selected students.
+        * Enrolled
+            This is a list of students who have been enrolled in the course. Moderator can reject enrolled students.
+        * Rejected
+            This is a list of students who have been rejected for enrollment in a course. Moderator can enroll rejected students.
+        * Deactivate/Activate Course
+            Clicking on this will deactivate or activate the course.
+        * Upload Users
+            Create and enroll users automatically by uploading a csv of the users. The mandatory fields for this csv are - **firstname, lastname, email**. Other fields like Institute, Department, Username, Password, Roll number and Remove field are optionals.
```

Comment:
This is just a confirmation.
So, the mandatory fields are exact as in headers, the other fields are not exact headers, I mean Roll Number in csv is roll_no, Username is username etc.
So, Is it fine?

Reply:
No, actually its confusing. I fixed it. Thanks